apps/nuxt3-ssr/package.json

```diff
@@ -8,7 +8,6 @@
     "dev": "nuxt dev",
     "generate": "nuxt generate",
     "preview": "nuxt preview",
-    "postinstall": "nuxt prepare",
```

Comment:
why is this removed as part of the expressions PR

Reply:
thanks for spotting this, this must be a merge error of some sort. Because I think nuxt3-ssr shouldn't have been touched.